source/support.html.erb

```diff
@@ -15,22 +15,18 @@ description: Help and support connecting to GovWifi
 
       <h2 class="heading-medium">User support</h2>
       <p>
-        Use the left-hand navigation bar to view our support guides for assistance in connecting to GovWifi for the first time.
-      </p>
-      <p>
-        If you experience any problems with your username or password,
```

Comment:
I don't think we should completely remove the support form.

I think for now, let's just push them to double check their credentials, or get password resets, but leave the user support as a "last effort" of sorts